### ISSUE TITLE
[Рейд] Плато Солнечного Колодца (ID 24882 - Бруталл)

### ISSUE THREAD

**Issue by Santaria94** [VirginTeam/WotlkCore#1189]
**Ссылки и комментарии:**

http://db.logon3.com/?spell=46394

**Что работает не так?**

Способность не усиливается со временем, тикает примерно одинаково на протяжении всей дистанции.

**Как должно работать?**

Из тактики на данного босса.
> Burn: Вешает на случайного игрока дот, тикающий 60 секунд. Дот работает как думфаер на Архимонде, но с точностью до наоборот - сначала он тикает мало, а под конец разгоняется и последний тик уже на **6400**. Дот имеет способность "перепрыгивать" на рядом стоящих игроков. Не диспелится, но его можно срезистить.

**Текущее обновление ядра:**

217

[//]: # (** Всё, что выделено спецальными символами, это базовая информация о шаблоне баг репорта, её не нужно удалять, она не попадёт в сам репорт и предназначена для удобства оформления репортов. **)
[//]: # (** Этот шаблон предназначен для всех репортов по игровым и серверным проблемам, не редактируйте его и не изменяйте без надобности **)
[//]: # (** Перед созданием репорта прочитайте основную информацию, как оформлять репорты и заголовки. **)
[//]: # (** Приоритеты и ярлыки назначаются только сотрудниками проекта и/или модераторами багтрекера **)
[//]: # (** Ни в коем случае не указывайте название игровой учётной записи или пароль от неё, вы можете оставить имя персонажа для обратной связи с вами **)
[//]: # (** Если вы хотите задать вопрос или уточнить что-то связанное с разработкой, присоединяйтесь к нашему каналу в Discord - https://discord.gg/A7kMHba **)

**Comment by Santaria94** [comment]
> The first 10 ticks will tick for 100, the next 11 will tick for 200, then 400 for 11 more, 800 for 11 more, 1600 for 11 more and finally the last six ticks will do 3200

**Comment by Plasmashield** [comment]
Еще, если не ошибаюсь, дота могла перенакладываться поверх уже тикающей. Скорее всего так быть не должно

**Comment by Santaria94** [comment]
Нет, она не перенакладывалась. Там именно фиксированное количество урона на протяжении всего действия доты.
